pkgs/by-name/am/amiberry/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  makeWrapper,
+  flac,
+  libmpeg2,
+  libmpg123,
+  libpng,
+  libserialport,
+  portmidi,
+  SDL2,
+  SDL2_image,
+  SDL2_ttf,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "amiberry";
+  version = "5.7.4";
+
+  src = fetchFromGitHub {
+    owner = "BlitterStudio";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
I didn't have this initially, but I looked at nixpkgs and saw this pattern used, assuming it was the common practice. Good you point out that it isn't! Cheers.